### PR TITLE
Update Russian and Ukrainian translations

### PR DESCRIPTION
**Pull request by Minoru**

### CODE REVIEW

**Comment by Minoru** [comment]
Ping @der-lyse, @dennisschagt; you usually take a look ;)

File: po/uk.po [hunk 1]
```diff
@@ -1723,9 +1721,9 @@ msgid "failed to parse binding"
 msgstr "не вдалося розібрати прив'язку клавіші"
 
 #: src/keymap.cpp:1028
-#, fuzzy, c-format
+#, c-format
 msgid "unknown context: %s"
-msgstr "невідома команда `%s'"
+msgstr "невідомий контекст %s"
```
Comment (der-lyse) [inline po/uk.po:1726]: Both the English and Russian messages contain a colon to separate the message prefix and the actual context value. So, I reckon the Ukrainian translation might want to have one, too. But maybe not.

**Review by der-lyse — APPROVED** [review]
Apart from the potentially missing colon, this looks alright to me. :-)

File: po/uk.po (hunk 1 above)
Comment (Minoru) [reply po/uk.po]: Yep, you're totally right. Thanks for spotting that!

**Pull request merged**